qpylib/json_qpylib.py

```diff
@@ -5,82 +5,68 @@
 import json
 from . import app_qpylib
 
-# A dictionary of jsonld context types mapped to the type name
 JSONLD_TYPES = {}
-
 KEY_CONTEXT = '@context'
 KEY_TYPE = '@type'
 KEY_ID = '@id'
 
 def register_jsonld_endpoints():
```

Comment:
Just checking, this is just refactored but functionally the same?

Reply:
Refactored with some corrections (I think). Wil wrote most of the JSONLD stuff originally, not sure anyone actually uses it.